test/it/constants.ts

```diff
@@ -0,0 +1,5 @@
+export const TIMEOUT = 100;
+export const SLOW = 100;
+export const RATHER_SLOW = 90;
```

Comment:
90 seems too close to 100, with only 10ms distance. Maybe use uniform distances between the various time points, like 50ms, 100ms, 150ms, 200ms.

Reply:
Currently trying out the limits. It seems that the timers on appveyor are running differently, so that node's nextTick() will be executed only every 50ms or so.

Reply:
Maybe they are running on the Escher cloud...

https://www.google.de/search?q=optical+illusion+escher&tbm=isch&tbo=u&source=univ&sa=X&ved=2ahUKEwjFjrPT75jfAhVR_aQKHRHdAjwQsAR6BAgCEAE&biw=1920&bih=1067

Reply:
The test seem to be working now. Maybe the event loop needs some time before it picks up faster running tasks...